support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -76,7 +76,7 @@ case object Paper extends Product {
   private def homeDelivery(productRatePlanId: ProductRatePlanId, productOptions: ProductOptions, description: String): ProductRatePlan[Paper.type] =
     ProductRatePlan(productRatePlanId, Monthly, HomeDelivery, productOptions, description, List(CountryGroup.UK))
 
-  private lazy val useDigitalVoucher = false
+  lazy val useDigitalVoucher = false
```

Comment:
doesn't need to be lazy given it's a boolean. although I realise that wasn't your change here!

Reply:
So should it be `val useDigitalVoucher = false`?